test/WebJobs.Extensions.Tests/Extensions/DocumentDB/Trigger/CosmosDBTriggerAttributeBindingProviderTests.cs

```diff
@@ -98,31 +88,29 @@ public static IEnumerable<ParameterInfo[]> GetParameters()
                     new[] { GetFirstParameter(type, "Func1") },
                     new[] { GetFirstParameter(type, "Func2") },
                     new[] { GetFirstParameter(type, "Func3") },
-                    new[] { GetFirstParameter(type, "Func4") },
-                    new[] { GetFirstParameter(type, "Func5") },
-                    new[] { GetFirstParameter(type, "Func6") }
+                    new[] { GetFirstParameter(type, "Func4") }
                 };
             }
         }
 
         private static class ValidCosmosDBTriggerBindigs
```

Comment:
It's likely because you're not a part of our team directly -- we don't let non-team members have access to our connection strings for security reasons. That's another reason to mock out `INameResolver` -- you can use Moq to make sure that you're asking for the `AzureWebJobsDocumentDBConnectionString` as the fallback without actually needing the environment variable set up.